pkg/cmd/setup.go

```diff
@@ -66,6 +69,11 @@ func setup(cmd *cobra.Command, args []string) {
 		utils.Log(valueFromEnvironmentNotice("DOPPLER_PROJECT"))
 		selectedProject = localConfig.EnclaveProject.Value
 	default:
+		if repoConfig.Setup.Project != "" {
+			selectedProject = repoConfig.Setup.Project
+			utils.LogDebug("Reading project from doppler.yaml")
```

Comment:
I think this should always be logged, rather than making it debug-only. This would be similar to line 55 above where we read values out of the environment. To see what I mean, run `DOPPLER_PROJECT=123 doppler setup`.